server/controller/assessment/getTableData.ts

```diff
@@ -13,32 +14,28 @@ export const getTableData = async (
     tableNames: Array<string>
   },
   client: BaseProtocol = DB
-): Promise<Array<{ tableName: string; data: TableData }>> => {
+): Promise<TableData> => {
   const { countryIso, assessmentName, cycleName, section, tableNames } = props
   if (!section) throw new Error('missing section')
 
+  const tables: Record<string, any> = {}
+  tableNames.forEach((tableName) => {
+    tables[tableName] = {}
+  })
+
   return client.tx(async (t) => {
     const assessment = await AssessmentRepository.read({ name: assessmentName }, t)
+    const cycle = assessment.cycles.find((cycle) => cycle.name === cycleName)
```

Comment:
these should be passed from api and must be got from AssessmentController.getOneWithCycle